appengine/pubsub/app.standard.yaml

```diff
@@ -17,7 +18,8 @@ instance_class: F2
 
 # [START gae_pubsub_yaml]
 env_variables:
-  PUBSUB_TOPIC: YOUR_TOPIC_NAME
+  #PUBSUB_TOPIC: YOUR_TOPIC_NAME
+  PUBSUB_TOPIC: sample-topic
```

Comment:
Did/will you update the docs accordingly, if necessary?

Reply:
Done. Thanks!